### PR TITLE
Update foundry-start-to-end.md

### PR DESCRIPTION
**Pull request by wuzhong-papermoon**
### Description/Original PRs

Translation for PR:
https://github.com/moonbeam-foundation/moonbeam-docs/pull/847
https://github.com/moonbeam-foundation/moonbeam-docs/pull/908

builders/build/eth-api/dev-env/foundry.md is already up to date

### Checklist

- [✅] If this requires removing old images from the `moonbeam-docs` repo, I have created a corresponding PR
- [✅] If this requires adding/updating/deleting redirects, I have created a corresponding PR in the `moonbeam-mkdocs` repo
- [✅] If this requires removing old variables from the `moonbeam-docs` repo, I have created a corresponding PR

### Corresponding PRs

Please link to any additional corresponding PRs here.

### CODE REVIEW

File: tutorials/eth-api/foundry-start-to-end.md [hunk 1]
```diff
@@ -385,7 +237,21 @@ forge test
 forge test -vvvv
 ```
 
-![Forking Tests in Foundry](/images/tutorials/eth-api/foundry-start-to-end/foundry-4.webp)
+--8<-- 'code/tutorials/eth-api/foundry-start-to-end/terminal/test4.md'
+
+That's it for testing! You can find the complete `Container.t.sol` and `MyToken.t.sol` files below:
```
Comment (eshaben) [inline tutorials/eth-api/foundry-start-to-end.md:242]: Can you please translate this line?

**Review by eshaben — COMMENTED** [review]
Thank you! Just one line to be translated, please!

File: tutorials/eth-api/foundry-start-to-end.md (hunk 1 above)
Comment (wuzhong-papermoon) [reply tutorials/eth-api/foundry-start-to-end.md:242]:
```suggestion
测试完成！完整的 `Container.t.sol` 和 `MyToken.t.sol` 文件如下：
```

**Review by wuzhong-papermoon — COMMENTED** [review]
Ah sorry about it.

**Review by eshaben — COMMENTED** [review]
Wow sorry I didn't even realize until now, but you created a new page in the tutorials section, but the linked original PRs in this PR description are for changes to the foundry page in the build section.

We don't want to add this page to the tutorials section. So can you just make sure that the build section page is up to date?

https://github.com/moonbeam-foundation/moonbeam-docs/pull/847/files#diff-486503116f9f0f2eaac893e5515b5aa270b5a948db271274e4f5604b645cc233

**Pull request closed without merging**